azimuth/modules/model_performance/metrics.py

```diff
@@ -183,8 +187,12 @@ def make_probabilities(self) -> np.ndarray:
         return probs
 
 
-class MetricsPerFilterModule(AggregationModule[AzimuthConfig]):
-    """Computes the metrics for each filter."""
+class MetricsPerFilterModule(AggregationModule[MetricConfig], ExpirableMixin):
+    """Computes the metrics for each filter.
+
+    Note: This module is an ExpirableMixin given that it should get recomputed if smart tags are
```

Comment:
"smart tags" or "proposed actions"?
```suggestion
    Note: This module is an ExpirableMixin given that it should get recomputed if proposed actions are
```

Reply:
Based on that, I reverted some of my changes (`AzimuthConfig` is still the scope of `MetricsPerFilter`). However, I removed from `MetricsPerFilter` the computation of the metrics per outcome, because it was never used, and per data action, because it was never used and it would not have worked anyhow, given it was not an expirable module.

Reply:
Thanks! The note comment is perfect. 👍